cpp/ps0003/main (3).cpp

```diff
@@ -1,16 +1,22 @@
-#include<iostream>
+#include <iostream>
 using namespace std;
+
 int main()
 {
+    // prompting user for number 2/2
+    int num;
+    cout << "enter your num \n";
+    cin >> x;
+
+    // checking if the number is odd or even 3/3
+    if (num >= 0)
+    {
+        if (num % 2 == 0)
+            cout << "even \n";
+        else
+            cout << "odd \n";
+    }
```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console